index.js

```diff
@@ -28,6 +29,15 @@ class ScrollTrigger {
   }
 
   calcBounds() {
+    // Element is hidden and not fixed
+    if (!this.el.offsetParent && typeof this.options.fixed !== 'undefined') {
```

Comment:
so what if something else decides to show the image, it won't load then, right?

Reply:
That's right. This is still an improvement as this would load right now nevertheless. Also the options.fixed overwrites this.

If you decide to load images on hover is probably safer to just append the images on said event rather than doing strange stuff.

Reply:
I think for the hover scenario, we shouldn't use something like scroll triggers

Reply:
Exactly. I was wrong on my initial spec. If you expect to manually make images appear then we just append them to the DOM which is same as lazy.